django_dynamic_fixture/ddf.py

```diff
@@ -380,7 +380,13 @@ def set_data_for_a_field(self, model_class, __instance, __field, persist_depende
         else:
             if self.debug_mode:
                 LOGGER.debug('%s.%s = %s' % (get_unique_model_name(model_class), __field.name, data))
-            setattr(__instance, __field.name, data) # Model.field = data
+            try:
+                setattr(__instance, __field.name, data) # Model.field = data
+            except ValueError:
+                if is_relationship_field(__field):
+                    setattr(__instance, "%s_id" % __field.name, data) # Model.field = data
+                else:
+                    raise
```

Comment:
Is this `raise` a good option for Python 2 and 3? It would be a good idea to use the `six.reraise`. Could you refactor that?